types/detox/index.d.ts

```diff
@@ -434,10 +434,12 @@ declare global {
     }
 }
 
-export { by, detox, device, element, waitFor };
+export { by, device, element, waitFor };
 
 // Not exporting the global `expect` from the top of the file here
 // because `expect` conflicts with the global `expect` of jest and
 // therefore exporting it doesn't work. The global `expect` is kept
 // for backwards compatibility though.
 export const expect: Detox.Expect<Detox.Expect<any>>;
+
+export default detox;
```

Comment:
I couldn't sort through the library's documentation to check this. Where is the default export defined in the source/documentation?

Reply:
The `module.exports` from detox's [index.js](https://github.com/wix/Detox/blob/master/detox/src/index.js#L87) matches the definition for `Detox.Detox`. There isn't located in the Detox documentation, however.

Reply:
I see - So I think may want `export = detox` as it's exported as a CommonJS module (assignment to `module.exports`)

`export default detox` is slightly different, it would correspond to a literal `export default detox` in the source.

Reply:
The corresponding import statement to `export = foo` would be `import foo = require('foo')` or `import * as foo from 'foo'`

Reply:
Thanks for the feedback. I'll need to update the Jest and Mocha adapters as well because they also are exported as CommonJS modules